src/dataflow-types/src/logging.rs

```diff
@@ -116,6 +117,13 @@ impl LogVariant {
                 .with_column("count", ScalarType::Int64.nullable(false))
                 .with_key(vec![0, 1, 2]),
 
+            LogVariant::Timely(TimelyLog::Messages) => RelationDesc::empty()
+                .with_column("channel", ScalarType::Int64.nullable(false))
+                .with_column("source_worker", ScalarType::Int64.nullable(false))
+                .with_column("target_worker", ScalarType::Int64.nullable(false))
+                .with_column("sent", ScalarType::Int64.nullable(false))
+                .with_key(vec![0]),
```

Comment:
I'm not sure if just `channel` is sufficient to be the key? it seems like `channel`s are themselves keyed on`(id, worker)` so I think the appropriate key here is `(channel, source_worker, target_worker)`

Reply:
That was it, oof. I think channel, source_worker is probably sufficient to uniquely identify a channel, but (channel, source_worker, target_worker) is probably more accurate. Will make it so.